### PR TITLE
Use wget again

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Use wget again as curl removes the trailing dot.

**Release note**:
```release-note
Revert wget->curl replacement
```

### CODE REVIEW

File: api/pkg/resources/apiserver/is-running.go [hunk 1]
```diff
@@ -8,6 +8,13 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+var cmdTpl = `
+while true;do
+    wget --timeout=1 --no-dns-cache --no-check-certificate %s/healthz && break
+    echo "waiting for apiserver..."
+    sleep 2
+done`
```
Comment (thz) [inline api/pkg/resources/apiserver/is-running.go]: may run without end. should abort after ~100 cycles?

Comment (mrIncompetent) [reply api/pkg/resources/apiserver/is-running.go]: Good point - It should die at some point

File: api/pkg/resources/apiserver/is-running.go [hunk 2]
```diff
@@ -8,6 +8,13 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+var cmdTpl = `
+while true;do
+    wget --timeout=1 --no-dns-cache --no-check-certificate %s/healthz && break
```
Comment (thz) [inline api/pkg/resources/apiserver/is-running.go]: curl image has no wget.
busybox wget has no `--no-dns-cache` nor `--no-check-certificates`.

So we would need a wget image ;)

Comment (mrIncompetent) [reply api/pkg/resources/apiserver/is-running.go]: I modified the curl image to include wget :P

File: api/pkg/resources/apiserver/is-running.go [hunk 3]
```diff
@@ -8,6 +8,16 @@ import (
...
 )
 
+var cmdTpl = `
+for (( i=1; i<=100; i++ ))
+do
+    echo "[${i}] Probing apiserver..."
+    wget --tries=1 --timeout=1 --no-dns-cache --no-check-certificate %s/healthz && break
+    sleep 2
+done
+exit 1
```
Comment (thz) [inline api/pkg/resources/apiserver/is-running.go:18]: This is exiting with failure on successful /healthz check, rigth?

File: api/pkg/resources/apiserver/is-running.go [hunk 4]
```diff
@@ -18,16 +28,12 @@ func IsRunningInitContainer(data resources.DeploymentDataProvider) (*corev1.Cont
 
 	return &corev1.Container{
 		Name:            "apiserver-running",
-		Image:           data.ImageRegistry(resources.RegistryQuay) + "/kubermatic/curl:v0.1",
+		Image:           data.ImageRegistry(resources.RegistryQuay) + "/kubermatic/curl:v0.2-dev-2",
```
Comment (thz) [inline api/pkg/resources/apiserver/is-running.go:31]: curl image is now including wget and bash too?

**Pull request closed without merging**